bin/start_fe.sh

```diff
@@ -60,7 +60,7 @@ source $STARROCKS_HOME/bin/common.sh
 # JAVA_OPTS
 # LOG_DIR
 # PID_DIR
-export JAVA_OPTS="-Xmx1024m"
+export JAVA_OPTS="-Xmx8G"
 export LOG_DIR="$STARROCKS_HOME/log"
```

Comment:
-Xmx8g?

wondering if we can do more in common.sh, e.g. check host mem, allocate half of total mem up to 8g at most by default.